pipelines/LSSTCam/DRP-FL.yaml

```diff
@@ -116,26 +101,6 @@ tasks:
     class: lsst.pipe.tasks.drpAssociationPipe.DrpAssociationPipeTask
     config:
       connections.finalVisitSummaryRefs: preliminary_visit_summary
-  skyCorr:
```

Comment:
`skyCorr` can't be removed here unless you make this pipeline inherit from `DRP.yaml`, or if you add `skyCorr` to `_ingredients/base-v2.yaml`